src/docs/development/accessibility-and-localization/accessibility.md

```diff
@@ -115,6 +132,16 @@ widgets to offer a dramatically more accessible experience.
 
 <iframe width="560" height="315" src="https://www.youtube.com/embed/bWbBgbmAdQs" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
+## Testing accessibility on web:
+
+You can debug accessibility by visualizing the semantic nodes created for your web app using the following command line flag in profile and relese modes:
```

Comment:
nit: (applies throughout the file) it seems this file is attempting to keep a certain line length (looks like 80, but I didn't count)

Reply:
Yup! The GitHub rule is to break at 80 and we often break much less than that.